src/reducer.js

```diff
@@ -60,6 +60,11 @@ export default function translationReducer(inputState = initialState, action = {
         state
       );
     }
+
+    case actions.SET_FORM_VALIDITY: {
+      const { form, valid } = action;
+      return state.setIn(['fields', form, '__valid?'], valid);
```

Comment:
this `__valid?` will be returned as `field` most probably. So when your form will have `email` and `password` you will get on submit {email, password, __valid?}.

pls test this scenario